src/components/fields/derived.js

```diff
@@ -196,6 +197,8 @@ export const AxesRange = connectToContainer(UnconnectedAxisRangeValue, {
   },
 });
 
+export const NumericOrDate = connectToContainer(UnconnectedNumericOrDate);
```

Comment:
Did you forget to delete this one?

Reply:
no...
this is the connected component I'm using everywhere..